test/fs/test_fs_js_api.c

```diff
@@ -37,6 +39,84 @@ int main() {
 #endif
     );
 
+    /********** test FS.truncate() **********/
+    EM_ASM(
+        FS.writeFile('truncatetest', 'a=1\nb=2\n');
+    );
+
+    struct stat s;
+    stat("truncatetest", &s);
+    assert(s.st_size == 8);
+
+    EM_ASM(
+        FS.truncate('truncatetest', 2);
```

Comment:
As with all these changes it would be good to look for other tests that already use this API, if any?   If we were previously disabling, or not running, those tests under WASMFS it would be good to enable them now.

Reply:
I didn't see anything explicitly calling `FS.truncate()` in the test dir, so I added them to `test_fs_js_api` for now. Are there any places besides the test directory that could potentially contain disabled tests?